doc/users/prev_whats_new/whats_new_1.4.rst

```diff
@@ -221,7 +221,6 @@ Added size related functions to specialized `.Collection`\s
 Added the ``get_size`` and ``set_size`` functions to control the size of
 elements of specialized collections (
 :class:`~matplotlib.collections.AsteriskPolygonCollection`
-:class:`~matplotlib.collections.BrokenBarHCollection`
```

Comment:
This shouldn't be removed from old docs, even though it doesn't exist any more, because it's a historical reflection of "what's new in 1.4". Mark it as code style instead:
```suggestion
``matplotlib.collections.BrokenBarHCollection``
```